script/default-site

```diff
@@ -12,7 +12,7 @@ workdir=$(pwd)
 echo "$0: creating new default site"
 if [[ "$(uname -s)" != "Linux" ]]
 then 
-  exec exe/jekyll new tmp/default-site
+   `exe/jekyll new tmp/default-site`
```

Comment:
Looking at the logic it seems that bit is meant to run on OS X. /cc @benbalter and @parkr who originally created it so they can further explain the logic behind it.